src/.formatted-files

```diff
@@ -0,0 +1,8 @@
+stat.h
+tweak.cpp
+tweak.h
+thinblock.cpp
+thinblock.h
+leakybucket.h
+parallel.cpp
+parallel.h
```

Comment:
you have a point, but what if I made a change on a file that is not on that list? Isn't the aim of this PR to validate/check the formatting of the submitted changes?